sdk/storage/azqueue/service_client_test.go

```diff
@@ -281,7 +281,7 @@ func (s *RecordedTestSuite) TestServiceListQueuesWithMetadata() {
 	for pager.More() {
 		resp, err := pager.NextPage(context.Background())
 		_require.Nil(err)
-		for _, queue := range resp.QueueItems {
+		for _, queue := range resp.QueuesList {
```

Comment:
QueueItems sound more intutive then QueueList (does not clearly tell what it means)

Reply:
I'm ok with either or here, @jhendrixMSFT what's your input on this?

Reply:
Is the suffix even necessary, can it just be `Queue`?

Reply:
@jhendrixMSFT so `Queues`? I'm ok with that as well.